app/helpers/fhir_helper.rb

```diff
@@ -800,6 +927,147 @@ def from_address_by_type_extension(element, address_type)
     address
   end
 
+  # Return an extension that captures all of the exposure risk factors for a monitoree
+  def to_exposure_risk_factors_extension(patient)
+    subextensions = [
+      to_risk_factor_subextension('contact-of-known-case', 'contact-of-known-case-id', patient.contact_of_known_case, patient.contact_of_known_case_id),
+      to_risk_factor_subextension('was-in-health-care-facility-with-known-cases', 'was-in-health-care-facility-with-known-cases-facility-name',
+                                  patient.was_in_health_care_facility_with_known_cases, patient.was_in_health_care_facility_with_known_cases_facility_name),
+      to_risk_factor_subextension('laboratory-personnel', 'laboratory-personnel-facility-name', patient.laboratory_personnel,
+                                  patient.laboratory_personnel_facility_name),
+      to_risk_factor_subextension('healthcare-personnel', 'healthcare-personnel-facility-name', patient.healthcare_personnel,
+                                  patient.healthcare_personnel_facility_name),
+      to_risk_factor_subextension('member-of-a-common-exposure-cohort', 'member-of-a-common-exposure-cohort-type',
+                                  patient.member_of_a_common_exposure_cohort, patient.member_of_a_common_exposure_cohort_type),
+      to_bool_extension(patient.travel_to_affected_country_or_area, 'travel-from-affected-country-or-area'),
+      to_bool_extension(patient.crew_on_passenger_or_cargo_flight, 'crew-on-passenger-or-cargo-flight')
+    ]
+
+    return nil if subextensions.all?(&:nil?)
```

Comment:
In reality I don't there's any functional difference between `nil` and `false` for the risk factors. Technically, the way it's implemented, `false` means a user has at some point toggled the value from true to false, but from the users' perspective a value of `nil` in the database _is_ false. And in excel export, we do include a value of "false" where the value in the record is actually `nil`. So I suggest removing this, treating `nil`s as `false`, and always including the `exposure-risk-factors` extension.

Reply:
Good point, I didn't realize we did that with export. I will update.

Reply:
Should be done in https://github.com/SaraAlert/SaraAlert/pull/1016/commits/b4c6ce032d9891fa042e18bc92fe30ae9985f497.